website/blog/2019-05-28-first-blog-post.mdx

```diff
@@ -0,0 +1,12 @@
+---
+slug: what-is-web-socket
+title: What is Web Socket?
+authors: [KenC]
+tags: [Azure Functions, Serverless]
+---
+
+import AsSeenOn from "@site/src/components/Blogs/AsSeenOn";
+
+<AsSeenOn />
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque elementum dignissim ultricies. Fusce rhoncus ipsum tempor eros aliquam consequat. Lorem ipsum dolor sit amet
```

Comment:
you need to remove all the place holders, since once the PR is merged to dev, it will automatically deploy

Reply:
This placeholder has been removed.